.pre-commit-config.yaml

```diff
@@ -26,20 +26,16 @@ repos:
         files: \.(json|json\.tpl)$
         args: [--autofix]
 
-  # pre-commit hook to verify commit messages are conventional commit compliant
```

Comment:
I can not reproduce this with the latest version, that's why I removed this comment.